doma-core/src/main/java/org/seasar/doma/jdbc/query/AutoInsertQuery.java

```diff
@@ -138,6 +150,21 @@ protected void prepareSql() {
       builder.cutBackSql(2);
     }
     builder.appendSql(")");
+  }
+
+  private void assembleUpsertSql(PreparedSqlBuilder builder, Naming naming, Dialect dialect) {
+    UpsertAssemblerContext context =
+        UpsertAssemblerContextBuilder.fromEntity(
+            builder,
+            entityType,
+            duplicateKeyType,
+            naming,
+            dialect,
+            idPropertyTypes,
+            targetPropertyTypes,
+            entity);
+    UpsertAssembler upsertAssemblerQuery = dialect.getUpsertAssembler(context);
```

Comment:
`upsertAssembler` is a more desirable variable name than `upsertAssemblerQuery`.